handbook/business-operations/business-operations.rituals.yml

```
@@ -142,6 +142,13 @@
   description: "Verify completion of any objective or outcome based bonus plans, and process payroll." 
   moreInfoUrl: "https://fleetdm.com/handbook/business-operations#run-us-commission-payroll" # TODO update linked process and add a new process that captures MBO payment
   dri: "jostableford" 
+-
+  task: "Review state filings for the previous quarter" #TODO tie this to a responsibility 
+  startedOn: "2024-07-19" 
+  frequency: "Quarterly"
+  description: "Verify that state filings have been successfully submitted for the previous quarter" 
+  moreInfoUrl: null 
```

```suggestion
  moreInfoUrl: 
```
Ok to leave this blank for now for consistency's sake.